src/capture.cpp

```diff
@@ -2295,27 +2299,17 @@ const char *locale = DEFAULT_LOCALE;
 #define MAXMEAN 134
                     int minAcceptableMean = MINMEAN;
```

Comment:
@EricClaeys 
if you do a function with a clean interface for the mean calculation (ZWO, RPiHQ) we could work better together (if wanted)

Reply:
@AndreasLMeg, I'm not sure I understand.  Do you mean move all the code that attempts to get to an acceptable mean into a function rather than as part of main() ?